### ISSUE TITLE
404 when connecting to Prismic repository

### ISSUE THREAD

**Issue by ktbee** [angeloashmore/gatsby-source-prismic#33]
I'm hoping this is user error and that you can point me in the right direction. I am getting this error whenever I run `gatsby develop`:

```
success delete html and css files from previous builds — 0.021 s
success open and validate gatsby-config — 0.550 s
success copy gatsby files — 0.022 s
success onPreBootstrap — 0.436 s
⠁ Starting to fetch data from Prismic
error Unexpected status code [404] on URL https://undefined.prismic.io/api/v2?access_token=[ACCESS_TOKEN]


  Error: Unexpected status code [404] on URL https://undefined.prismic.io/api/v2?access_token=[ACCESS_TOKEN]
```

The access token is correct, but obviously the repository URL is not. How can I make sure the repository name is populated? Below is my gatsby-config.js file:

```
const dotenv = require('dotenv');
const dotenvParseVariables = require('dotenv-parse-variables');

const env = dotenv.config();
dotenvParseVariables(env.parsed);

module.exports = {
  plugins: [
    {
      resolve: 'gatsby-source-prismic',
      options: {
        repository: 'repo-name',
        accessToken: process.env.PRISMIC_KEY,
        linkResolver: doc => {
          return '/' + doc.uid;
        },
        htmlSerializer: ({ node, key, value }) => (
          (type, element, content, children) => {
            // Your HTML serializer
          }
        )
      }
    },
  ],
};
```

**Comment by angeloashmore** [comment]
Hi @ktbee, thanks for pointing this out. `repository` should be `repositoryName` in the config options.

My mistake; the README is wrong. I’ll update the example.

Thanks!

**Comment by ktbee** [comment]
That solved it, thank you! 🎉

**Comment by bzhr** [comment]
Hey, just as a reminder, this issue is closed, while the error in the docs isn't fixed. It can be pretty disheartening for a beginner to not be able to start the project. 

Regards

**Comment by angeloashmore** [comment]
Thanks, just published `v1.0.1` with the typo fix. 👍

**Comment by lucasreppewelander** [comment]
i have this config

```
{
	resolve: 'gatsby-source-prismic',
	options: {
		repositoryName: 'corporate-x', // (REQUIRED, replace with your own)
		accessToken: token_here',
		linkResolver: doc => {
			return '/' + doc.uid;
		},
		htmlSerializer: ({ node, key, value }) => (
			(type, element, content, children) => {
				// Your HTML serializer
			}
		)
	}
}
```

but i still get the error telling me:
`Unexpected status code [404] on URL https://corporate-x.prismic.io/api/v2?access_token`
